src/Endpoint/ConfigCreate.php

```diff
@@ -5,33 +5,33 @@
 class ConfigCreate extends \Docker\API\Runtime\Client\BaseEndpoint implements \Docker\API\Runtime\Client\Endpoint
 {
     /**
+     * 
      *
-     *
-     * @param null|\Docker\API\Model\ConfigsCreatePostBody $requestBody
+     * @param null|\Docker\API\Model\ConfigsCreatePostBody $requestBody 
      */
     public function __construct(?\Docker\API\Model\ConfigsCreatePostBody $requestBody = null)
     {
         $this->body = $requestBody;
     }
     use \Docker\API\Runtime\Client\EndpointTrait;
-    public function getMethod() : string
+    public function getMethod(): string
     {
         return 'POST';
     }
-    public function getUri() : string
+    public function getUri(): string
     {
         return '/configs/create';
     }
-    public function getBody(\Symfony\Component\Serializer\SerializerInterface $serializer, $streamFactory = null) : array
+    public function getBody(\Symfony\Component\Serializer\SerializerInterface $serializer, $streamFactory = null): array
     {
         if ($this->body instanceof \Docker\API\Model\ConfigsCreatePostBody) {
-            return array(array('Content-Type' => array('application/json')), $serializer->serialize($this->body, 'json'));
+            return [['Content-Type' => ['application/json']], $serializer->serialize($this->body, 'json')];
```

Comment:
same for arrays - if we didn't have all the array to `[]` would that generate a smaller diff?